terraform/addons/byo-firehose-logging-destination/target-account/firehose.tf

```diff
@@ -32,7 +32,8 @@ data "aws_iam_policy_document" "firehose_policy" {
     actions = ["logs:PutLogEvents"]
     resources = [
       "arn:aws:logs:${data.aws_region.current.id}:${data.aws_caller_identity.current.account_id}:log-group:/aws/kinesisfirehose/${var.firehose_results_name}:*",
-      "arn:aws:logs:${data.aws_region.current.id}:${data.aws_caller_identity.current.account_id}:log-group:/aws/kinesisfirehose/${var.firehose_status_name}:*"
+      "arn:aws:logs:${data.aws_region.current.id}:${data.aws_caller_identity.current.account_id}:log-group:/aws/kinesisfirehose/${var.firehose_status_name}:*",
+      "arn:aws:logs:${data.aws_region.current.id}:${data.aws_caller_identity.current.account_id}:log-group:/aws/kinesisfirehose/${var.firehose_audit_name}:*"
```

Comment:
if `var.firehose_audit_name` is empty string be default do you see an issue with this IAM document? @zwinnerman-fleetdm @rfairburn ?